controllers/courses.js

```diff
@@ -0,0 +1,136 @@
+const mongoose = require('mongoose')
+const { Users, Roles } = require('../models/users')
+const Courses = require('../models/courses')
+
+const courseController = {};
+
+/**
+ * @param {Object} req 
+ * @param {Object} res 
+ * @param {Function} next 
+ * @description Fetches all courses' data 
+ */
+courseController.getAll = async (req, res, next) => {
+    try {
+        const allCourses = await Courses.find()
```

Comment:
This is a wrong method of filtering the data, it should be handled in the query itself.
`const allUsers = await Users.find({ is_deleted: false });`